app/src/main/java/edu/drexel/lapcounter/lapcounter/frontend/temp/LapCounterServiceTest.java

```diff
@@ -305,4 +310,29 @@ protected void onRestart() {
         super.onRestart();
         log("onRestart");
     }
+
+    @Override
+    public boolean onKeyUp(int keyCode, KeyEvent event) {
```

Comment:
If someone is listening to music and does volume up/down, will their music go up or down as well?
Also, is this a feature we want to have in the final, or is it for test use?

Reply:
it will go up or down, I'm told this is a feature for test; it's currently in one of the test screens.